### ISSUE TITLE
Debug hammer error

### ISSUE THREAD

**Issue by KiwiTrek** [FeroXx07/Donkey-Kong#13]
When toggling on hammer again by holding button, multiple music loops start happening. Also when moving in one direction, player's sprite bugs.

**Comment by FeroXx07** [comment]
Yes, it has to do with the timing of the hammer, since it is one time pick up item.
